### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by annashcherbina**

### CODE REVIEW

File: chrombpnet/helpers/hyperparameters/param_utils.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def filter_edge_regions(peaks_df, bw, width, peaks_bool):
     chrom_to_sizes = bw.chroms()
     filtered = []
     for i, r in peaks_df.iterrows():
-        if r['start'] + r['summit'] + width//2 > chrom_to_sizes[r['chr']] :
+        if r['start'] + r['summit'] + width//2 > chrom_to_sizes.get(r['chr'], 0) :
```
Comment (panushri25) [inline chrombpnet/helpers/hyperparameters/param_utils.py:24]: why is this change needed?

Comment (annashcherbina) [reply chrombpnet/helpers/hyperparameters/param_utils.py:24]: @austintwang made this change in this commit: https://github.com/kundajelab/chrombpnet/commit/ea661207bf3faa767c4ab4d0d1cd91dee6afb527
I'm not sure why it is needed, @austintwang can you please comment on what led to this change? Thanks!
